src/main/java/org/terasology/anatomy/component/AnatomyPartTag.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.anatomy.component;
+
+import com.google.common.collect.Lists;
+import org.terasology.network.Replicate;
+import org.terasology.reflection.MappedContainer;
+
+import java.util.List;
+
+@MappedContainer
+public class AnatomyPartTag {
+    /**
+     * Id of the anatomy part. Useful for multi-word names.
+     */
+    public String id = "";
+
+    /**
+     * Name of the anatomy part.
+     */
+    @Replicate
+    public String name = "";
+
+    @Replicate
+    public List<String> effects = Lists.newArrayList();
```

Comment:
I'm not catching what you mean here with the events being received simultaneously, could you give an example, maybe on Slack sometime when we get a chance?

And do you think the base Anatomy system actually needs to know? If the AnatomyX systems react to `AnatomyPartImpactedEvent` and trigger effects there, wouldn't they be attached to the entity rather than tracked by Anatomy at that point? With the AnatomyX systems and their Components tracking when something needs to deactivate.

On Slack `InGameHelp` came up as an example of the modifyable "pattern" but I'm not sure about the details, probably poke the others in #content that seem to know about it :-)